samples/mpu/mpu_stack_guard_test/sample.yaml

```diff
@@ -4,6 +4,8 @@ tests:
   test:
     tags: apps
```

Comment:
this particular sample really need to be converted into a proper test.
i think there's a bug open on it somewhere.
nothing to change in your patch here, just whining

Reply:
agree, will address in another PR